internal/server/data/migrations.go

```diff
@@ -56,7 +57,21 @@ func migrations() []*migrator.Migration {
 	}
 }
 
+//go:embed schema.sql
+var schemaSQL string
+
 func initializeSchema(db *gorm.DB) error {
+	if db.Dialector.Name() == "sqlite" {
```

Comment:
Do we still need this sqlite automigrate where some of the migrations wont work on sqlite anyway?

Reply:
Probably not, but it wasn't much work and I'm trying to keep some support for sqlite where its easy.